bouffalo-hal/src/psram.rs

```diff
@@ -0,0 +1,72 @@
+//! Pseudo Static Random Access Memory.
+
+use crate::glb;
+use core::ops::Deref;
+use volatile_register::RW;
+
+/// Pseudo Static Random Access Memory registers.
+#[repr(C)]
+pub struct RegisterBlock {
+    /// Basic configuration register.
+    pub basic_config: RW<u32>,
+    _reserved0: [u8; 0x1C],
+    /// Psram configuration register.
+    pub psram_config: RW<u32>,
+    _reserved1: [u8; 0xDC],
+    /// Phy configuration register.
+    pub phy_config: [RW<u32>; 21],
+}
+
+/// Initializes the PSRAM.
+#[inline]
+pub fn init_psram<
+    PSRAM: Deref<Target = RegisterBlock>,
+    GLB: Deref<Target = glb::v2::RegisterBlock>,
+>(
+    psram: &PSRAM,
+    glb: &GLB,
+) {
```

Comment:
```suggestion
pub fn init_psram(
    psram: &RegisterBlock,
    glb: &glb::v2::RegisterBlock,
) {
```

Will this work as well? (BTW, please run `cargo fmt` before commit)

Reply:
done